### ISSUE TITLE
any way pull names or specify sensors as labeled in Govee app? ( way to correlate Govee app sensor names to devices)

### ISSUE THREAD

**Issue by saiajin** [asednev/homebridge-plugin-govee#30]
**Is your feature request related to a problem? Please describe:**
I have mulitiple sensors and they seem to come in as names like GVH5075C68A  but I cant fina a correlation in the Govee app so I having a hard time adding multiple sensors and differentiating them

**Describe the solution you'd like:**
possibly a way to correlate the sensor name as they are labeled in the Govee app or another method to tell them apart

**Describe alternatives you've considered:**
Tried looking for these IDs in the Govee app but cant find them

**Additional context:**

**Comment by asednev** [comment]
Thank you for submitting this suggestion. 

**TLDR**: I would gladly accept a contribution if someone were to open a PR.

---
I have not been able to find this information in the bluetooth announcements made by Govee devices. I suspect that device name might be maintained locally on the app, and not on the device. Govee devices do broadcast another announcement that I have not been able to decrypt. It might contain additional information.

**Comment by saiajin** [comment]
Hello! I agree cant seem to find any correlation or unique way to identify each sensor....I have 5 so its a real pain.....Sorry I am still learning here..How does a PR work?   Are you saying for now its not possible? Thank you for all you do!!!

**Comment by saiajin** [comment]
BTW I am testing this on HOOBS 4 Beta now... seems to be working... just getting this in logs...


4/3/2021, 2:58:21 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH50755A3F@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:21 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH5075C68A@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:21 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH5075C68A@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:21 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH5075C68A@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:23 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH50755A3F@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:23 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH50755A3F@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:23 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH50755A3F@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:25 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH5075C68A@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:25 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH5075C68A@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:25 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH5075C68A@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:33 PM
Plugin Govee Bridge
WARNING
[Plugin Govee Bridge E6D8@GVH5075C68A@@Current Relative Humidity] characteristic was supplied illegal value: number 290 exceeded maximum of 100
4/3/2021, 2:58:33 PM
Plugin Govee Bridge
WARNIN

**Comment by asednev** [comment]
> BTW I am testing this on HOOBS 4 Beta now... seems to be working... just getting this in logs...
> 
I moved this over to a new issue #33

**Comment by pattullo** [comment]
> Thank you for submitting this suggestion.
> 
> **TLDR**: I would gladly accept a contribution if someone were to open a PR.
> 
> I have not been able to find this information in the bluetooth announcements made by Govee devices. I suspect that device name might be maintained locally on the app, and not on the device. Govee devices do broadcast another announcement that I have not been able to decrypt. It might contain additional information.

Regarding this - as a potential work around might it be possible to simply allow for naming of the devices based on bluetooth MAC in the interim? Not exactly ideal of course but users can reach the same effective end-state.
